omod/src/main/resources/config.xml

```diff
@@ -60,21 +64,6 @@
 	</messages>
 	<!-- /Internationalization -->
 
-	<advice>

```

Comment:
I suspect these cannot/should not be removed, and I'd hope that they are invoked optionally if one does not have the atom feed module installed then they have no impact.

Reply:
Makes sense